### ISSUE TITLE
Velocity implementation

### ISSUE THREAD

**Issue by SeptyIsHere** [AuthMe/AuthMeBungee#71]
Can you make AuthmeBungee work for this standalone of bungeecord: https://www.velocitypowered.com/downloads

**Comment by 7wOv6ySCjo** [comment]
Yes !! we need AuthMeBungee for Velocity by the official team of AuthMe. @sgdc3

**Comment by LucidAPs** [comment]
Please add Velocity support.

**Comment by 7wOv6ySCjo** [comment]
Yes.

**Comment by 7wOv6ySCjo** [comment]
Please, we need a port to VelocityPowered Proxy of this. Urgently.

**Comment by ghost** [comment]
Bump

**Comment by 7wOv6ySCjo** [comment]
Bump.

**Comment by 7wOv6ySCjo** [comment]
.

**Comment by x9nico** [comment]
Hey!
I found a fork of AuthmeBungee for Velocity ! => https://github.com/MTM123/AuthMeVelocity

**Comment by x9nico** [comment]
If you want the jar of AuthmeVelocity, there is ! @7wOv6ySCjo @MacacoLew @skbeh @SeptyIsHere
(It supports only 1.1.x versions)

[AuthmeVelocity.zip](https://github.com/AuthMe/AuthMeBungee/files/5972863/AuthmeVelocity.zip)

**Comment by 7wOv6ySCjo** [comment]
> Hey!
> I found a fork of AuthmeBungee for Velocity ! => https://github.com/MTM123/AuthMeVelocity

Beware, the plugin has false positives with "Authentication required".
I myself have asked for such a fork to be made.

**Comment by a8underscore** [comment]
this would be great

**Comment by Jakubk15** [comment]
we definitely need this.

**Comment by rautamiekka** [comment]
Absolutely !

There are plugins which support Velocity, so you can take a look at their code if necessary. Some of which:

- [LuckPerms](https://github.com/lucko/LuckPerms) has a separate Velocity binary.
- [SkinsRestorerX](https://github.com/SkinsRestorer/SkinsRestorerX) uses a single binary for all supported platforms.
- [Plan](https://github.com/plan-player-analytics/Plan) uses a single binary for all supported platforms.

All of those share a common base code within the plugin, while the platforms have their specific codes separated.

[More plugins on the Forum](https://forums.velocitypowered.com/c/plugins/plugin-releases/6).

[Jenkins for 3.x.x](https://ci.velocitypowered.com/job/velocity-3.0.0/).

**Comment by ghost** [comment]
up

**Comment by LucidAPs** [comment]
up!

**Comment by Neeonn** [comment]
up

**Comment by compieter-gh** [comment]
up

**Comment by krusic22** [comment]
Already exists in the velocity-support branch.

**Comment by LucidAPs** [comment]
> Already exists in the velocity-support branch.

The branch from 2 years ago with no current builds?

**Comment by krusic22** [comment]
2021 was last year.
And yes, you have to build it yourself.

**Comment by LucidAPs** [comment]
That makes it really inaccessible for (I would argue) 99% of users, including my self. If we could at least get a jar to test it, that would be great.

**Comment by krusic22** [comment]
Directly copied from Discord:

Here: https://posili.me/randstuff/AuthMeBungee-2.2.0-velocity.jar
and: https://posili.me/randstuff/AuthMe-5.6.0-velocity.jar
Both updated to latest master (at the time), so you have all the latest changes + velocity.
Of course, 0 support with that.

**Comment by LucidAPs** [comment]
From Discord? The authme discord?
Also thanks for the links.

**Comment by krusic22** [comment]
The AuthMe Discord linked in the readme.

**Comment by hex3l** [comment]
Builds for that branch can be found here, if you run into issues/need infos on configuration steps for velocity you could try and send me a DM on discord.
https://ci.hex3l.net/job/AuthMereloaded/
I am pretty busy at the moment and this implementation needs at least one more feature and  polishing before being merged so it could take some time.

**Comment by 7ooki** [comment]
使用snap
